openedx/features/lti_course_tab/tab.py

```diff
@@ -34,7 +34,40 @@ class LtiCourseLaunchMixin:
     }
     DEFAULT_ROLE = 'Student'
 
+    def _get_pii_lti_parameters(self, course: CourseBlock, request: HttpRequest) -> Dict[str, str]:
+        """
+        Get LTI parameters that contain PII.
+
+        Args:
+            course (CourseBlock): CourseBlock object.
+            request (HttpRequest): Request object for view in which LTI will be embedded.
+
+        Returns:
+            Dictionary with LTI parameters containing PII.
+        """
+        lti_config = self._get_lti_config(course).lti_config
+        if lti_config.version != lti_config.LTI_1P1:
+            return {}
+        ask_to_send_username = lti_config.get('ask_to_send_username', False)
+        ask_to_send_email = lti_config.get('ask_to_send_email', False)
```

Comment:
@xitij2000, the `version` and `LTI_1P1` are one level "above" the `lti_config` attribute. I did the following on the sandbox and it works correctly, but the `lti_config.lti_config` is confusing, so please find a better name for this.

```suggestion
        lti_config = self._get_lti_config(course)
        if lti_config.version != lti_config.LTI_1P1:
            return {}
        ask_to_send_username = lti_config.lti_config.get('ask_to_send_username', False)
        ask_to_send_email = lti_config.lti_config.get('ask_to_send_email', False)

```